pyproject.toml

```diff
@@ -1,6 +1,5 @@
 [build-system]
 requires = [
-  "setuptools == 41.0.0",  # See https://github.com/ansible/molecule/issues/2350
```

Comment:
This must be transformed into >= in order to revert our previous pinning. It should be enough.